site/layouts/shortcodes/ods-recommendation-alert.html

```diff
@@ -0,0 +1,13 @@
+{{- /*
+Usage: `ods-recommendation-alert`
+*/ -}}
+
+<div class="bd-ods-alert alert alert-info">
+  <span class="alert-icon"><span class="visually-hidden">Info</span></span>
+  <div>
+    <div class="alert-heading mb-1 h4">
+      Recommendation of Orange Design System
```

Comment:
Missing end point ?

Reply:
In my opinion no end point for info alert, because here it is not a sentence but a title.
In the case of danger alert, since we have "more info" after, we need a point (dot) to separate with "more info"